central/notifiers/cscc/client/client.go

```diff
@@ -42,6 +43,7 @@ type Config struct {
 }
 
 func (c *Config) postURL(findingID string) string {
+	log.Infof("ROX-7273 finding:%s source:%s", findingID, c.SourceID)
```

Comment:
the log object is not initialized so this will error. But good thing is that we have a reference to it within the config object.

So change to `c.Logger.Infof`

Reply:
ok sure let me change that